api/v1/helper.go

```diff
@@ -801,3 +802,39 @@ func NetFilterMatch(netFilter string, netValue string) (isMatch bool) {
 
 	return netFilterResult[0][1] == netValueResult[0][1] && netFilterResult[0][2] == netValueResult[0][2]
 }
+
+// MaxUnavailable calculate the max number of unavailable nodes to represent the number of nodes
+// we can drain in parallel
+func (s *SriovNetworkPoolConfig) MaxUnavailable(numOfNodes int) (int, error) {
+	// this means we want to drain all the nodes in parallel
+	if s.Spec.MaxUnavailable == nil {
```

Comment:
I think the default should be 100%. We should not treat -1 as being the same as 100%.

Reply:
can we return value from numOfNodes in this case?

Reply:
can we have a race if we do that and a new node is added?